tests/integration-tests/tests/dashboard/test_dashboard/test_dashboard/pcluster.config.yaml

```diff
@@ -0,0 +1,40 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    VpcId: {{ vpc_id }}
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+Scheduling:
+{% if {{ scheduler }} == "awsbatch" %}
+  Awsbatch:
+    Queues:
+      - Name: queue1
+        Networking:
+          SubnetIds:
+            - {{ public_subnet_id }}
+        ComputeResources:
+          - Name: compute_resource1
+            InstanceType: {{ instance }}
+{% else %}  # scheduler == "Slurm"
```

Comment:
The code is the same for batch and slurm you can avoid to duplicate it and just do:
```python
  {% if {{ scheduler }} == "awsbatch" %}Awsbatch:{% else %}Slurm:{% endif %}
```
but in future we need to fix scheduler to be camelcase and directly use scheduler value
```
Scheduling:
  {{ scheduler }}:
    Queues:
```

Reply:
I was thinking we may want to config the MinCount/MaxCount for slurm and MinVcus and MaxVcpus for awsbatch. Now I change it to {% if {{ scheduler }} == "awsbatch" %}Awsbatch:{% else %}Slurm:{% endif %} and I add a # FIXME there for using camelcase in the future